pulp_ansible/app/models.py

```diff
@@ -45,44 +43,68 @@ class Meta:
         unique_together = ("namespace", "name")
 
 
+class Tag(Model):
+    """A model representing a Tag.
+
+    Fields:
+
+        name (models.CharField): The Tag's name.
+    """
+
+    name = models.CharField(max_length=32, unique=True, editable=False)
```

Comment:
Why only 32 chars?

Reply:
I figured they wouldn't be longer, but now it seems short. What should it be?

Reply:
Actually 32 seems fine.

Reply:
Or let's do 64 to be safe?

Reply:
may want to encourage shorter than 64 tags, maybe keep 32 for now?